libs/common/src/assets/i18n/en.json

```diff
@@ -983,6 +984,153 @@
 	"synonyms.table.main-word": "Main word",
 	"synonyms.table.search": "Search a synonym",
 	"synonyms.table.synonyms": "Synonyms",
+	"tasks-automation.bach-to-tasks": "Back to tasks",
+	"tasks-automation.badge.beta": "Beta",
+	"tasks-automation.badge.optional": "Optional",
+	"tasks-automation.cta.new-task": "New task",
+	"tasks-automation.cta.see-archive": "See Archive",
+	"tasks-automation.errors.counting-labeled-resources": "An error occurred while loading the number of resources labeled with selected label set.",
```

Comment:
I know the message is accurate, but it is hard to understand, maybe:
`An error occurred while counting the resources labeled with the selected label set.`